x/fees/genesis.go

```diff
@@ -17,13 +17,22 @@ func InitGenesis(
 	k.SetParams(ctx, data.Params)
 
 	for _, fee := range data.Fees {
+		// prevent storing the same address for deployer and withdrawer
+		if fee.DeployerAddress == fee.WithdrawAddress {
+			fee.WithdrawAddress = ""
+		}
+
 		contract := common.HexToAddress(fee.ContractAddress)
 		deployer := sdk.MustAccAddressFromBech32(fee.DeployerAddress)
 		withdrawal := sdk.MustAccAddressFromBech32(fee.WithdrawAddress)
```

Comment:
this will result in an invalid address if the address is empty. We should set the address

```
		var withdrawal sdk.AccAddress
		if fee.DeployerAddress != fee.WithdrawAddress {
			withdrawal = sdk.MustAccAddressFromBech32(fee.WithdrawAddress)
		}
```